packages/common/src/hooks/useArtistCoinMessageHeader.ts

```diff
@@ -28,13 +28,10 @@ export const useArtistCoinMessageHeader = ({
     return null
   }
 
-  let artistCoinSymbol
-  let artistCoin
+  let ticker
   if (coin) {
-    artistCoin = coin
-    // Get symbol directly from the coin data
-    artistCoinSymbol = artistCoin.ticker
+    ticker = `$${coin.ticker}`
```

Comment:
does this work correctly for audio

Reply:
nobody will be sending blasts with audio tho right